WordPress/src/main/res/values/strings.xml

```diff
@@ -4010,6 +4010,59 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="blogging_prompts_onboarding_remind_me">Remind me</string>
     <string name="blogging_prompts_onboarding_card_prompt">Cast the movie of your life.</string>
 
+    <!-- WordPressEditor module -->
```

Comment:
Adding the `tools:ignore="UnusedResources"` here (as well as that comment) to all the entries that got previously merged from the WordPressEditor module, so that the linter doesn't error on those strings (not realizing they might not be used by the `main/src/` code but might still be used by the subproject)